packages/nimble-components/src/table-column/base/models/column-internals.ts

```diff
@@ -163,14 +174,27 @@ export class ColumnInternals<TColumnConfig> {
     @observable
     public currentSortDirection: TableColumnSortDirection = TableColumnSortDirection.none;
 
-    public constructor(options: ColumnInternalsOptions) {
+    public readonly validator: TColumnValidator;
+
+    public constructor(options: ColumnInternalsOptions<TColumnValidator>) {
         this.cellRecordFieldNames = options.cellRecordFieldNames;
         this.cellViewTemplate = createCellViewTemplate(options.cellViewTag);
         this.groupHeaderViewTemplate = createGroupHeaderViewTemplate(
             options.groupHeaderViewTag
         );
         this.delegatedEvents = options.delegatedEvents;
         this.sortOperation = options.sortOperation ?? TableColumnSortOperation.basic;
+        this.validator = options.validator;
+
+        const notifier = Observable.getNotifier(this.validator);
+        notifier.subscribe(this);
```

Comment:
~~Not sure if there is an effective perf difference, but subscribe takes an optional property name you want to monitor right?~~

edit: not needed if removing prop in other comment